app/src/main/java/edu/uc/groupProject/topten/ui/main/MainViewModel.kt

```diff
@@ -20,50 +23,153 @@ import edu.uc.groupProject.topten.Service.StrawpollService
 class MainViewModel : ViewModel() {
     var list:MutableLiveData<ArrayList<ListItem>> = MutableLiveData<ArrayList<ListItem>>()
     var listService: ListService = ListService()
+    private var firestore : FirebaseFirestore = FirebaseFirestore.getInstance()
```

Comment:
Added all logic for database handling here.

Reply:
Good.